### PR TITLE
Feat/lint

### PR DESCRIPTION
**Pull request by dolanmiu**
Fixes https://github.com/Cosium/dry-dry/issues/6 and https://github.com/Cosium/dry-dry/issues/4

I have added my tslint file with 2 added rules:

`quotemark` and `arrow-parens`

I don't mind which quote mark to use, double or single, as long as its consistent

arrow-parens, i like enforcing to always use parens in whatever case, to keep consistency, but your opinion may be different to mine :P 

so:
```ts
doStuff((hello) => {})
doTwoStuff((hello, world) => {})
```

rather than:

```ts
doStuff(hello => {}) // special one param case
doTwoStuff((hello, world) => {})
```

But it's up to you

### CODE REVIEW

**Comment by dolanmiu** [comment]
Have a look at these rules, we can talk here which ones to drop or add

**Comment by reda-alaoui** [comment]
I am find with all of the rules.
Once Travis passes, we are good to go ;)

**Comment by dolanmiu** [comment]
I have fixed all linting and style, code looks much more consistent now

Few comments I have added, especially the use of `any`, but that can be done later

**Comment by reda-alaoui** [comment]
In the last version of the patch, quote were forced to single.
The current version use double.
Could you revert to single quote?

**Comment by dolanmiu** [comment]
I made it to double cos the majority was actually double, and only one file was single

so I thought maybe you intended it to all be double

will make it single

File: src/dependency-resolver.ts [hunk 1]
```diff
@@ -1,46 +1,38 @@
-import {Cli} from "./cli";
-import {DryDependencies} from "./dry-dependencies";
+import { Cli } from "./cli";
+import { DryDependencies } from "./dry-dependencies";
```
Comment (dolanmiu) [inline src/dependency-resolver.ts]: Like here, see what I mean

**Comment by dolanmiu** [comment]
I have made it to single quotes now

File: src/dry-package.ts [hunk 2]
```diff
@@ -1,65 +1,71 @@
...
-    private static readonly PACKAGE_DRY_JSON = 'package-dry.json';
-
-    private constructor(private readonly dependencyResolver: DependencyResolver,
-                        private readonly location: string,
-                        private _content: DryPackageContent & any) {
-
-    };
-
-    static readFromDisk(dependencyResolver: DependencyResolver): DryPackage {
+    public static readFromDisk(dependencyResolver: DependencyResolver): DryPackage {
```
Comment (reda-alaoui) [inline src/dry-package.ts]: Sorry I didn't see that earlier.
Is it possible to have all attributes always at the top of the class and all methods come after?

Comment (dolanmiu) [reply src/dry-package.ts]: Done, overwritten the default `member-ordering` rule, which states that static functions have to be before constructor

I personally think everything `static` should come before the constructor, cos it is not instance specific, but again, its my opinion :P

**Comment by reda-alaoui** [comment]
Good work ;)

**Pull request merged**